ambry-api/src/main/java/com.github.ambry/store/MessageInfo.java

```diff
@@ -21,8 +21,8 @@
  */
 public class MessageInfo {
 
-  public static final short ACCOUNT_ID_DEFAULT_VALUE = -1;
-  public static final short CONTAINER_ID_DEFAULT_VALUE = -1;
+  public static final short ACCOUNT_ID_LEGACY_VALUE = -1;
```

Comment:
Merged the blob props patch - can this be removed now ? (and all other places where this is defined if anywhere else - like `IndexValue`)